content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -1109,25 +1103,25 @@ For example:
 
 For example, the following query returns `User` that meet the following criteria:
 
-* No posts with more than 100 views
-* All posts have more than 50 likes
+- No posts with more than 100 views
+- All posts have more than 50 likes
 
 ```ts
 const users = await prisma.user.findMany({
   where: {
     posts: {
       none: {
         views: {
-          gt: 100
-        }
+          gt: 100,
+        },
       },
       every: {
         likes: {
-          lte: 50
-        }
-      }
-    }
-  }
+          lte: 50,
+        },
+      },
+    },
+  },
```

Comment:
I guess the commas are one of the prettier opinionations here?

Reply:
It's because of this rule: https://github.com/prisma/docs/blob/fad88c6f110219f4d2f567961036efd01e7d4704/.prettierrc#L6